src/test/java/com/google/api/codegen/testdata/discoveries/ruby/ruby_storage.v1.json.baseline

```diff
@@ -1045,6 +1045,11 @@ request_body = Google::Apis::StorageV1::ComposeRequest.new
 # TODO: Assign values to desired members of the `request_body` object.
 
 
+# TODO: Specify destination to download media content. For more information, see:
```

Comment:
Two thoughts:
(a) Can we hint at it inline?
```
// service.compose_object(..., mediaDestination=newMediaDestination().setLocation())
```
(b) would it make sense to always show the destination variable, but simply have the value be an empty placeholder in the snippet? (And, if anything, say in the destination variable declaration `TODO: fill in the desired destination. If you don't want to download media content, you may omit this parameter entirely in the function call below`.) Thoughts?

If neither of these are easy or you disagree, I am fine deferring this discussion until later.